flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
+  "require-important-update": true
```

Comment:
this is not needed either.

Reply:
should I remove the entire flathub.json? isn't t needed to tell flathub that my app runs on X86_64 only?

Reply:
why should it run only on x86_64?

Reply:
because I'm not able to test on different arch currently and I don't want to ship things I can't test.

Reply:
so no one will ever be able to test it.

Reply:
never say never :) not today, does not mean, never.

Reply:
ah ok removing it. fixed.